### ISSUE TITLE
Not Working When After Apps Build to APK

### ISSUE THREAD

**Issue by YudeMgoy** [hua1995116/google-translate-open-api#15]
Works fine when debugging but after build its not working anymore

**Comment by hua1995116** [comment]
Can you print some log information about the web after you package it into apk？

**Comment by YudeMgoy** [comment]
I catch the error and it says error with status code 403

**Comment by sittingbool** [comment]
Getting a CORS error when running in electron. Could be the same reason

**Comment by hua1995116** [comment]
I found a problem with the user-agent, I have now upgraded to 1.3.4 , which is fine. Using isUserAgent: false to solve the problem, I tested in electron@9.0.5 It works well. I think it might be the same problem in apps.
```
const result = await translate([`I'm fine. And you?`,`I'm ok.`], {
        tld: "cn",
        to: "zh-CN",
        isUserAgent: false
});
```

**Comment by hua1995116** [comment]
Here's a demo of the repo.
https://github.com/hua1995116/google-translate-open-api-demo/tree/master/electron

**Comment by sittingbool** [comment]
Hi, I could solve it with that fix: I set electrons web security to false in a previous hotfix and it gave me that userAgent error. Now that you fixed that as well it works like a charm but I still have to disable electrons websecurity in development mode because it acts as a server and requests want to be originated from localhost. but that seems to be an unavoidable problem with electron.

**Comment by hua1995116** [comment]
@sittingbool You don't need to close the SCP. You can open a separate whitelist.  You can look at this case.
https://github.com/hua1995116/google-translate-open-api-demo/blob/master/electron/index.html#L6-L7

**Comment by YudeMgoy** [comment]
its still error in my app

**Comment by hua1995116** [comment]
Can you show your code? Then show more error details

**Comment by YudeMgoy** [comment]
![IMG_20200708_090200](https://user-images.githubusercontent.com/45964922/87002790-8e295880-c1e4-11ea-8fac-19efca5bc4c8.jpg)

Its my code, but i have no idea what's the error message. I try with try and catch but it return nothing

**Comment by phuongpt** [comment]
I have same issue, API returns `Request failed with status code 403` on React Native Android release build.
![Screen Shot 2020-07-09 at 6 09 29 PM](https://user-images.githubusercontent.com/4443583/87032853-6b606980-c20f-11ea-944f-b02aceb1138e.png)

**Comment by YudeMgoy** [comment]
> I have same issue, API returns `Request failed with status code 403` on React Native Android release build.
> ![Screen Shot 2020-07-09 at 6 09 29 PM](https://user-images.githubusercontent.com/4443583/87032853-6b606980-c20f-11ea-944f-b02aceb1138e.png)

That's same as me, i use react native too and also have that error

**Comment by hua1995116** [comment]
Since I haven't developed an RN, I tested it in my spare time, and here's the code I tested, which is in both dev and release are working fine. Can you provide me with more and more complete information, as I have no way to troubleshoot it at this time?
Here are my code.
https://github.com/hua1995116/google-translate-open-api-demo/blob/master/react-native/reactNativeProject/App.js  
This is Android release APK.
[app-release.apk.zip](https://github.com/hua1995116/google-translate-open-api/files/4901658/app-release.apk.zip)
@phuongpt @YudeMgoy

**Comment by YudeMgoy** [comment]
> Since I haven't developed an RN, I tested it in my spare time, and here's the code I tested, which is in both dev and release are working fine. Can you provide me with more and more complete information, as I have no way to troubleshoot it at this time?
> Here are my code.
> https://github.com/hua1995116/google-translate-open-api-demo/blob/master/react-native/reactNativeProject/App.js
> This is Android release APK.
> [app-release.apk.zip](https://github.com/hua1995116/google-translate-open-api/files/4901658/app-release.apk.zip)
> @phuongpt @YudeMgoy

Is tld must be filled?

**Comment by hua1995116** [comment]
https://github.com/hua1995116/google-translate-open-api/blob/master/src/translate.ts#L28
No, There will be restrictions on accessing some sites in China, so adopted the use of `cn` and defaulted to `com`.

**Comment by hua1995116** [comment]
@YudeMgoy Are you using my apk to access properly?

**Comment by YudeMgoy** [comment]
> @YudeMgoy Are you using my apk to access properly?

yes, i have tried your apk and i see the code is same the differences just at the language code and tld.
Could you try for me use language code "id" instead

**Comment by hua1995116** [comment]
So maybe it's because of the regional restrictions on the different domains of google. I upgraded to version `1.3.5` and then tried using the Google Translate suffix for your current region. You can try.
Best wishes!

**Comment by hua1995116** [comment]
Did you solve the problem by changing the domain extension? @YudeMgoy

**Comment by YudeMgoy** [comment]
I try with default code you give, but its still not solved yet. Nothing error but the code wont run i think, my state variable still with default value.

**Comment by YudeMgoy** [comment]
Its strange, my first time try is there is error code 503, but now its doesnt give me anything even the code wont run

**Comment by phuongpt** [comment]
@hua1995116 I guess your apk was built in debug mode. I built my app with your code by Appcenter in release mode
![Screenshot_20200713-110322](https://user-images.githubusercontent.com/4443583/87269981-f932a200-c4f8-11ea-9bbc-6b60478f08c0.png)

**Comment by OSD-LuanNguyen** [comment]
Same here. I get release apk and get the same error.

**Comment by hua1995116** [comment]
Does this problem still exist? @OSD-LuanNguyen @YudeMgoy @sittingbool

**Comment by YudeMgoy** [comment]
> Does this problem still exist? @OSD-LuanNguyen @YudeMgoy @sittingbool

Yap it still exist on my app

**Comment by hua1995116** [comment]
I used xcode to test again, but still can't reproduce your error. Do you think my configuration is consistent with yours? I did not reproduce this error on iphone x and iphone 8 plus. The demo address is https://github.com/hua1995116/google-translate-open-api-demo/tree/master/react-native/reactNativeProject
![image](https://user-images.githubusercontent.com/12070073/95044803-f79d0280-0712-11eb-9061-8e7291c6c786.png)


> > Does this problem still exist? @OSD-LuanNguyen @YudeMgoy @sittingbool
> 
> Yap it still exist on my app

**Comment by YudeMgoy** [comment]
> I used xcode to test again, but still can't reproduce your error. Do you think my configuration is consistent with yours? I did not reproduce this error on iphone x and iphone 8 plus. The demo address is https://github.com/hua1995116/google-translate-open-api-demo/tree/master/react-native/reactNativeProject
> ![image](https://user-images.githubusercontent.com/12070073/95044803-f79d0280-0712-11eb-9061-8e7291c6c786.png)
> 
> > > Does this problem still exist? @OSD-LuanNguyen @YudeMgoy @sittingbool
> > 
> > 
> > Yap it still exist on my app

Maybe you can try on android project, then build the app first, i mean a full build not debuging build
